bin/docker_start

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+require 'uri'
+require 'active_record'
+
+if ENV['AUTO_MIGRATE'] == '1'
+  pg_uri = URI.parse(ENV['DATABASE_URL'])
+  port = pg_uri.port || 5432
+
+  # check that we can connect to Postgres
+  connected = false
+  60.times do
+    begin
+      puts "Attempting DB connection at #{pg_uri.host}:#{port} with user #{pg_uri.user}"
+      ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
+      connected = true
+      break
+    rescue Exception
+      sleep(1.second)
```

Comment:
Can we reraise this exception on the last attempt to get a good error message? Also should we catch StandardError?

Reply:
changed